changelog/unreleased/fix-statuscode-move-denied.md

```diff
@@ -0,0 +1,5 @@
+Bugfix: Fixed wrong status code when moving a file to a denied path
+
+We fixed a bug when the status code was 403 instead of 502 when moving a file to a denied path to be compatible with oc10.
```

Comment:
IMO you are correcting the status code that was 502, but now returns 403 (forbidden).
```suggestion
We fixed a bug when the status code 502 was returned when moving a file to a denied path. Status code 403 (forbidden) is now returned to be compatible with oc10.
```
Maybe the above text describes the bugfix better?